src/aui-form-validator/tests/unit/js/tests.js

```diff
@@ -190,6 +190,48 @@ YUI.add('aui-form-validator-tests', function(Y) {
             Y.Assert.isFalse(validator.hasErrors());
         },
 
+        /*
+         * Check if validator correctly validates dates in pt-br format dd/MM/yyyy
+         * @tests AUI-1940
+         */
+          'test date format': function() {
```

Comment:
Hi @thiago-rocha,  the issue was solved on the jsp code on LPS-56585 with https://github.com/liferay/liferay-portal-ee/commit/3da5ba4ae0f8d228a6693abb61a8e9bd80aaec6b